lib/shopify-cli/version.rb

```diff
@@ -1,3 +1,3 @@
 module ShopifyCli
-  VERSION = "1.10.0"
+  VERSION = "1.10.1"
```

Comment:
Generally we don't bump the version in PRs, since we might batch a few together in a release. I'm [updating](https://github.com/Shopify/shopify-app-cli/pull/1222) the PR template to make this clearer.